torch/distributions.py

```diff
@@ -168,14 +174,20 @@ class Normal(Distribution):
     Args:
         mean (float or Tensor or Variable): mean of the distribution
         std (float or Tensor or Variable): standard deviation of the distribution
+        reparameterized (bool): whether to use reparameterization trick
```

Comment:
The `reparameterized` docs should probably live a `Properties:` or `Attributes:` section.